examples/userapi/subdomains.ipynb

```diff
@@ -0,0 +1,966 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Subdomains tutorial"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This tutorial is designed to introduce users to the concept of subomains in Devito and how to utilize them within simulations for a variety of purposes.\n",
```

Comment:
typo: subdomains

this whole file perhaps needs to go through a spell checker?

Reply:
Ta, fixed.